src/allotropy/parsers/thermo_fisher_nanodrop_eight/nanodrop_eight_parser.py

```diff
@@ -0,0 +1,272 @@
+                        device_control_document=[
+                            UltravioletAbsorbancePointDetectionDeviceControlDocumentItem(
+                                device_type="absorbance detector",
+                                detector_wavelength_setting=TQuantityValueNanometer(
+                                    value=260
+                                ),
+                            )
+                        ]
+                    ),
+                    absorbance=TQuantityValueMilliAbsorbanceUnit(
+                        _get_float(data, row, "A260")
+                    ),
+                )
+            )
+        a280_col = "A280"
+        if a280_col not in data.columns and "A280 10mm" in data.columns:
+            a280_col = "A280 10mm"
+        if _get_str(data, row, a280_col):
+            measurement_docs.append(
+                UltravioletAbsorbancePointDetectionMeasurementDocumentItems(
+                    measurement_identifier=str(_get_str(data, row, "A280 uuid")),
+                    sample_document=SampleDocument(
+                        sample_identifier=str(_get_str(data, row, "Sample ID"))
+                        if _get_str(data, row, "Sample ID")
+                        else "NA",
+                        well_plate_identifier=_get_str(data, row, "Plate ID"),
+                        location_identifier=_get_str(data, row, "Well"),
+                    ),
+                    # capture concentration on the A280 measurement document if the experiment type is
+                    # something other than DNA or RNA
+                    mass_concentration=_get_concentration(
+                        _get_float(data, row, str(self._get_concentration_col(data))),
+                        _get_str(data, row, "Units"),
+                    )
+                    if "NA" not in str(_get_str(data, row, "NA Type"))
+                    and self._get_concentration_col(data)
+                    else None,
+                    device_control_aggregate_document=UltravioletAbsorbancePointDetectionDeviceControlAggregateDocument(
+                        device_control_document=[
+                            UltravioletAbsorbancePointDetectionDeviceControlDocumentItem(
+                                device_type="absorbance detector",
+                                detector_wavelength_setting=TQuantityValueNanometer(
+                                    value=280
+                                ),
+                            )
+                        ]
+                    ),
+                    absorbance=TQuantityValueMilliAbsorbanceUnit(
+                        _get_float(data, row, a280_col)
+                    ),
+                )
+            )
+
+        return measurement_docs
+
+    def _get_concentration_col(self, data: pd.DataFrame) -> Optional[str]:
+        for col in data.columns:
+            if col.lower() in ["conc.", "conc", "concentration"]:
```

Comment:
missed this yesterday -- is this just a best guess based on files you've seen so far? Seems a tad hacky/hardcoded.

Reply:
It's probably ok to keep these, but if so let's extract a constant so this is more visible.

Reply:
yeah, this is mostly test driven since I don't have an official file spec.